gen/suites/transfer/main.go

```diff
@@ -155,4 +157,67 @@ func main() {
 			Func: failTransferUnknownSenderUnknownReceiver,
 		},
 	)
+
+	g.MessageVectorGroup("system_receiver",
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-system-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.SystemActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-init-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.InitActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-reward-actor",
+				Version: "v1",
+			},
+			Func:  transferToSystemActor(builtin.RewardActorAddr),
+			Mode:  ModeLenientAssertions,
+			Hints: []string{schema.HintIncorrect},
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-cron-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.CronActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-storage-power-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.StoragePowerActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-storage-market-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.StorageMarketActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-verified-registry-actor",
+				Version: "v1",
+			},
+			Func: transferToSystemActor(builtin.VerifiedRegistryActorAddr),
+		},
+		&MessageVectorGenItem{
+			Metadata: &Metadata{
+				ID:      "to-burnt-funds-actor",
+				Version: "v1",
+			},
+			Func:  transferToSystemActor(builtin.BurntFundsActorAddr),
+			Mode:  ModeLenientAssertions,
+			Hints: []string{schema.HintIncorrect},
+		},
```

Comment:
Gotta sort this out as we should be asserting the actual balance taking into account gas burnt during message processing.